docs/src/components/Header.astro

```diff
@@ -15,7 +15,7 @@ let { navigation = null } = Astro.props
     class="sticky top-0 z-50 px-4 py-3 border-b border-b-fair-pink dark:border-b-slate-800 transition duration-500 sm:px-6 lg:px-8 bg-cream dark:bg-[#171527]"
 >
     <div
-        class="w-full max-w-8xl mx-auto flex flex-wrap gap-5 items-center justify-between"
+        class="w-full max-w-8xl mx-auto flex flex-wrap gap-5 items-center justify-between sm:flex-nowrap"
```

Comment:
This is not needed

Reply:
Remove the `flex-wrap` instead?

Reply:
I'd rather keep the `flex-wrap`.

Reply:
Do you have another fix for the last screenshot where it displays on two lines from 768px to 772px wide

Reply:
I just checked that specific viewport width range and indeed the wrapping does occur. We can go along with your approach!